wforce/wforce-replication.hh

```diff
@@ -46,18 +46,20 @@ public:
   void setNumSiblingThreads(unsigned int num_threads) { d_num_sibling_threads = num_threads; }
   GlobalStateHolder<vector<shared_ptr<Sibling>>>& getSiblings() { return d_siblings; }
   void replicateOperation(const ReplicationOperation& rep_op);
+  void setEncryptionKey(const std::string& key) { d_key = key; }
+  std::string getEncryptionKey() { return d_key; }
```

Comment:
```suggestion
  std::string getEncryptionKey() const { return d_key; }
```

Reply:
> Are the builder changes intended to be included?
> Lacking the big picture, a few nits and a question.

The big picture is that weakforced is used as a submodule in another project, so the whole WforceReplication class was created to reduce code duplication in that project